### PR TITLE
Add documentation about legacy cb-auto files

### PR DESCRIPTION
**Pull request by bmw**
Fixes https://github.com/certbot/certbot/issues/8812 by implementing the plan Erica and I discussed there.

### CODE REVIEW

File: letsencrypt-auto-source/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Legacy letsencrypt-auto files
+
+certbot-auto and letsencrypt-auto were two names for the same self updating
```
Comment (ohemorange) [inline letsencrypt-auto-source/README.md:3]:
```suggestion
`certbot-auto` and `letsencrypt-auto` were two names for the same self-updating
```

File: letsencrypt-auto-source/README.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+shell script that wrapped Certbot. Old versions of the script continue to rely
+on letsencrypt-auto and letsencrypt-auto.sig in this directory to download and
```
Comment (ohemorange) [inline letsencrypt-auto-source/README.md:5]:
```suggestion
on pulling `letsencrypt-auto` and `letsencrypt-auto.sig` from this directory hosted on Github to download and
```

**Review by ohemorange — CHANGES_REQUESTED** [review]
other than nits, lgtm!

**Review by ohemorange — APPROVED** [review]

**Comment by warent** [comment]
Hi folks, thanks for this.

Can we enhance this documentation to help noobs like myself understand what the migration plan is?

i.e. I'm trying to run an old bash script that depends on files that no longer exist here, and I have no idea what I should be doing instead to fix the script.

**Comment by bmw** [comment]
@warent, was this your own shell script and was it doing something like expecting to find `certbot-auto` or `letsencrypt-auto` in the root of this repository?

I don't think we were expecting (m)any people to have custom code making that assumption so we don't have any specific documentation for that and probably aren't going to add any unless we see more demand. If you were using our scripts directly before though, their output should have told you what to do which is to go to https://certbot.eff.org/ and get updated instructions for how to install Certbot on your system.

**Comment by warent** [comment]
Thanks for the quick reply @bmw, it's a script I found in this tutorial https://dev.to/zenika/how-to-use-secured-wildcard-custom-domains-on-appengine-1a9b

It looks like they somehow used this for their Docker image https://github.com/Zenika/gcloud-certbot/blob/main/Dockerfile#L12

It's pretty niche though I'm not the only one wondering about it https://github.com/Zenika/gcloud-certbot/issues/1

**Comment by warent** [comment]
all good @bmw I ended up getting it working. Basically just had to delete any reference to certbot-auto and everything ran as expected https://github.com/montfont/gcloud-certbot

**Pull request merged**